dotnet/eng/Version.props

```diff
@@ -4,6 +4,7 @@
         <AzureOpenAIVersion>1.0.0-beta.15</AzureOpenAIVersion>
         <SemanticKernelVersion>1.7.1</SemanticKernelVersion>
         <SemanticKernelExperimentalVersion>1.7.1-alpha</SemanticKernelExperimentalVersion>
+        <SemanticKernelAgentsVersion>1.10.0-alpha</SemanticKernelAgentsVersion>
```

Comment:
You can update `SemanticKernelExperimentalVersion` to `1.10.0-alpha` directly (And SemanticKernelVersion to `1.10.0` as well